ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,23 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - Cosmic Studio
+ - Defense Advanced Research Projects Agency (DARPA) Defense Sciences Office
+ - Office of Naval Research (ONR)
+ - National Aeronautics and Space Administration (NASA)
+ - National Science Foundation (NSF)
+
+Programs
+--------
+ - NSF 125908, AGS-1651393
+ - Naval Research Laboratory N00173191G016 and N0017322P0744
```

Comment:
@rstoneback @jklenzing @JonathonMSmith please check to ensure these funding sources are correct for this package.  Can confirm that NSF 125908, NRL/ONR/DARPA are correct.